src/pages/index.js

```diff
@@ -147,50 +132,73 @@ updateAvatarButton.addEventListener("click", () => {
 });
```

Comment:
```js

updateAvatarButton.addEventListener("click", () => {
  avatarPopup.open();
});
```

* After submitting the avatar button is active http://joxi.ru/E2pZKygHjW3Epr  . You haen't corrected my remark https://github.com/Marcus3278/se_project_aroundtheus-main/pull/5#discussion_r1612381911 


It should be


```js
updateAvatarButton.addEventListener("click", () => {
  avatarPopup.open();
  updateAvatarFormValidator.toggleButtonState()  // disable the button
});

```